tools/clang/test/DXC/min16vector_enabled.hlsl

```diff
@@ -0,0 +1,19 @@
+// Validate that the offsets for a min16float vector is correct if 16-bit types are enabled
+// RUN: %dxc -T cs_6_5 -Od -Zi -Qembed_debug -enable-16bit-types %s | FileCheck %s
+
+// CHECK-NOT: {{.*}}!DIDerivedType{{.*}}name: "x"{{.*}}offset: {{.*}}
```

Comment:
Why's this one CHECK-NOT?

Reply:
The first element in the vector doesn't have an offset (is 0) so I'm double checking that there's no offset field!